eslint.config.js

```diff
@@ -54,8 +32,35 @@ module.exports = {
   },
   overrides: [
     {
-      "files": ["test/**/*.ts"],
-      "rules": {
+      files: ['*.ts'],
```

Comment:
Why create a new override block for ts files?

Reply:
The examples are in JavaScript, so if we want to share the configuration between the packages and the examples, we the typescript rules to only be enabled for `ts` files.